packages/core/src/components/sections/Alert/Alert.tsx

```diff
@@ -1,37 +1,41 @@
-import CommonAlert from 'src/components/common/Alert'
-
-import { Components, Props } from 'src/components/sections/Alert/Overrides'
+import { ReactNode } from 'react'
 
-const { Icon } = Components
+import CommonAlert from 'src/components/common/Alert'
+import {
+  Alert as AlertWrapper,
+  Icon,
+} from 'src/components/sections/Alert/Overrides'
 
 export interface AlertProps {
   icon: string
   content: string
   link: {
-    text: string
+    // It is only ReactNode when overridden as such
+    text: string | ReactNode
     to: string
   }
   dismissible: boolean
 }
 
 // TODO: Change actionPath and actionLabel with Link
```

Comment:
@ArthurTriis1 I think we remove this TODO right? Or we still need to do something about this Link?